### PR TITLE
Upgrade To NVD3 1.8.1 And d3.js 3.5.5

### PR DESCRIPTION
**Pull request by thomas-tosoni**
No more local nvd3. nvd3 and d3 are global.
Start doubleLineChart template code cleanup / indent.
Switch 'clicks' 'impressions' legend.

### CODE REVIEW

**Comment by thomas-tosoni** [comment]
Do not merge until X axis fix.

File: app/src/core/common/charts.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-define(['./module', 'nv.d3', 'nvd3-templates/doubleLineChart', 'd3'], function (module, nv, ignore, d3) {
```
Comment (dduponchel) [inline app/src/core/common/charts.js]: The `nv` and `d3` variables are used below.

File: app/src/nvd3-templates/doubleLineChart.js [hunk 2]
```diff
@@ -1,627 +1,631 @@
-/* global nv */
```
Comment (dduponchel) [inline app/src/nvd3-templates/doubleLineChart.js]: JSHint says that `nv` is still a global variable.

File: app/src/nvd3-templates/doubleLineChart.js [hunk 3]
```diff
@@ -1,627 +1,631 @@
...
+     * Private Variables
+     */
+
+    var showTooltip = function (e, offsetElement) {
+      var left = e.pos[0] + ( offsetElement.offsetLeft || 0 ),
+        top = e.pos[1] + ( offsetElement.offsetTop || 0),
+        x = xAxis.tickFormat()(lines1.x()(e.point, e.pointIndex)),
+        y = (e.series.right ? yAxis2 : yAxis1 ).tickFormat()(lines1.y()(e.point, e.pointIndex)),
+        content = tooltip(e.series.key, x, y, e, chart);
+      console.log("SHOW TOOLTIP: ", content);
```
Comment (dduponchel) [inline app/src/nvd3-templates/doubleLineChart.js]: Forgotten debug ?

**Comment by thomas-tosoni** [comment]
Can be merged. Fix for x axis will be in the next pull request.

**Pull request merged**